sigstore-java/src/test/java/dev/sigstore/tuf/TufClientTest.java

```diff
@@ -141,8 +144,108 @@ public void testRootUpdate_metaFileTooBig()
     var client = createTimeStaticTufClient(localStore);
     try {
       client.updateRoot();
-      fail();
+      fail("MetaFileExceedsMaxException expected as 2.root.json is larger than max allowable.");
     } catch (MetaFileExceedsMaxException e) {
+      // expected
+    }
+  }
+
+  @Test
+  public void testTimestampUpdate_throwMetaNotFoundException() throws IOException {
+    setupMirror("remote-timestamp-not-present");
+    var client = createTimeStaticTufClient(localStore);
+    assertThrows(
```

Comment:
I think you can short form this to (if you want):
assertThrows(MetaNotFoundException.class, client::updateTimestamp)